src/server/generic_family.cc

```diff
@@ -786,6 +786,49 @@ OpStatus OpMove(const OpArgs& op_args, string_view key, DbIndex target_db) {
   return OpStatus::OK;
 }
 
+// OpMove touches multiple databases (op_args.db_idx, target_db), so it assumes it runs
+// as a global transaction.
+// TODO: Allow running OpMove without a global transaction.
+OpStatus OpCopy(const OpArgs& op_args, string_view key, DbIndex target_db) {
```

Comment:
`Copy` is actually a non-trivial operation that requires very good understanding of Dragonfly architecture.

This code  won't work when keys reside on different thread shards. If you wish to learn more about dragonfly architecture and its transactional framework, please read the docs under docs/ folder.

Reply:
I'll go through the docs and share an one-pager for this feature. Do you recommend any other resources/docs i can look at in general to understand this better? 

Do you have any timeline in mind for this feature? I'm new to this domain, so will take a few weeks to deliver this.

Reply:
do you have any suggested order for reading files in the docs folder?

Reply:
df-share-nothing.md and transaction.md are good candidates